File: plugins/OrchidForms/src/main/java/com/eden/orchid/presentations/FormsModel.java
package com.eden.orchid.presentations;

import lombok.Getter;
import lombok.Setter;

import javax.inject.Inject;
import javax.inject.Singleton;
import java.util.Map;

@Singleton
public class FormsModel {

    @Getter @Setter
    private Map<String, Form> forms;

    @Inject
    public FormsModel() {
    }

    void initialize(Map<String, Form> forms) {
        this.forms = forms;
    }

}

File: plugins/OrchidForms/src/main/kotlin/com/eden/orchid/search/model/FormsModel.kt
package com.eden.orchid.search.model

import javax.inject.Inject
import javax.inject.Singleton

@Singleton
class FormsModel @Inject
constructor() {

    lateinit var forms: MutableMap<String, Form>

    fun initialize(forms: MutableMap<String, Form>) {
        this.forms = forms
    }

}
